File: Python/uberX.py
from car import Car


class UberX(Car):
    brand = str
    model = str

    def __init__(self, license, driver, passengers, brand, model):
        super().__init__(license, driver, passengers)
        self.brand = brand
        self.model = model

    def printDataCar(self):
        super().printDataCar()
        print("Brand: {}".format(self.brand))
        print("Model: {}".format(self.model))

File: Java/UberX.java
package Java;

class UberX extends Car {
    String brand;
    String model;

    public UberX(int id, String license, Account driver, int passengers, String brand, String model) {
        super(id, license, driver, passengers);
        this.brand = brand;
        this.model = model;
    }
}